addOns/pscanrules/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Fixed
+- Refactored CookieLooselyScopedScanRule to comply with the latest RFC standards and streamline the loosely scoped cookie check (Issue 8863).
```

Comment:
Better to use what the users see here than the class name 😀 
`Loosely Scoped Cookie`

https://www.zaproxy.org/docs/alerts/90033/